packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -5,7 +5,9 @@ const logger = require('../utils/logger');
  */
 class StatsGroup extends GroupHelper {
     static validOptions() {
-        return ['none', 'errors-only', 'minimal', 'normal', 'detailed', 'verbose', 'errors-warnings'];
+        let validArrayString = ['none', 'errors-only', 'minimal', 'normal', 'detailed', 'verbose', 'errors-warnings'];
+        let validArrayObject = [{ verbose: true }, { verbose: true, stats: 'verbose' }];
```

Comment:
We can have more valid array when it is object

Reply:
should i go for it? It would be more validated as well.